notebooks/Advanced_Hierarchical_Calibration.ipynb

```diff
@@ -0,0 +1,692 @@
+      "text/plain": [
+       "<Figure size 600x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = pyplot.subplots(dpi=100)\n",
+    "\n",
+    "for batch_id, y_obs in dataset.items():\n",
+    "    ax.scatter(time, y_obs, label=batch_id)\n",
+    "ax.legend(loc=\"lower right\")\n",
+    "ax.set(\n",
+    "    xlim=(0, None),\n",
+    "    ylim=(0, None),\n",
+    "    xlabel=\"time   [s]\",\n",
+    "    ylabel=\"readout   [a.u.]\",\n",
+    "    title=\"\"\"experimental data\\n$P_0=0, S_0=5\\ \\mathrm{mM}$\"\"\"\n",
+    ")\n",
+    "pyplot.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b513352",
+   "metadata": {},
+   "source": [
+    "## Building the Bayesian model\n",
+    "For analyzing our data, we should think about how it was generated.\n",
+    "For every bit of information about the experimental setup we have to specify a _prior probability_ distribution to be used in the model.\n",
+    "\n",
+    "We started with an initial product concentration of exactly zero ($P_0=0\\ mM$).\n",
+    "\n",
+    "The reaction was started with 5 mM of substrate, but it's was hard to handle, so we might have gotten the concentration wrong by ~5 %.\n",
+    "\n",
+    "$$ S_0 \\sim \\mathrm{LogNormal}(log(5), 0.05) $$\n",
```

Comment:
Is 0.05 as sigma in LogNormal equal to a 5 % error? Isn't that different because of the log transformation?

Reply:
Not exactly because it's a natural log, but it's also more of a matter of definition. Do you want the 5th percentile to be 0.95x lower? Or 1 standard deviation? Or something else?

As long as the number is not too high, it's a good rule of thumb.

Reply:
Yep, I just wanted to prevent criticism from picky reviewers ;)